sirenity/code/handler.py

```diff
@@ -0,0 +1,43 @@
+import ast
+import os
+
+allowed_modules = ["math"]
+
+
+class CodeHandler:
+    """A class used to handle code submissions from users"""
+
+    pass
+
+    def __init__(self, code: str) -> None:
+        self.code = code
+        pass
```

Comment:
```suggestion
    def __init__(self, code: str) -> None:
        self.code = code
```
`pass` is not necessary here